### ISSUE TITLE
Planning v4

### ISSUE THREAD

**Issue by duyluonglc** [duyluonglc/lucid-mongo#124]
- [ ] Add JSON API serializer to Lucid serializers list.
- [ ] Typescript support
- [ ] Support mongodb 4.0

**Comment by HapLifeMan** [comment]
Very good release, I can't wait!

Do you have any plan to support `lookup`?
And do you know when the v4 will be available?

Thanks for your amazing job 😄

**Comment by lcnvdl** [comment]
Hi! When is the release date?

**Comment by HapLifeMan** [comment]
It seems that the repo isn't active as it were anymore 😕

**Comment by lcnvdl** [comment]
I agree. It's a shame. I'm using Adonis JS with MongoDB, and it seems to be this the only compatible alternative

**Comment by HapLifeMan** [comment]
We're strongly considering to fork this repo and maintain/develop in our own.

**Comment by valmirphp** [comment]
@HapLifeMan you can count on me!

**Comment by duyluonglc** [comment]
Seem `adonis-lucid` is updating new version with many improvements and be rewritten in ts. I'll start this version soon when it release

**Comment by valmirphp** [comment]
nice,

create a list of issues, tagged with v4.
I can help you with some issues.

This project is very good, I think it is one of the best package to work with Adonis and Mongo.

**Comment by jntagadiad** [comment]
waiting for the v4 :)

**Comment by DeVoresyah** [comment]
I wish I can help contribute, but I don't understand at all. So I just can pray the v4 will release soon

**Comment by josepuma** [comment]
Any news on this?

**Comment by tridungle** [comment]
It's so great! I'm waiting for you

**Comment by EduardoReolon** [comment]
Is there any news?

Lucid-mongo fits perfectly with Adonis, better than mongoose. I could try to help if I was smart enough :).

I tried to use transaction, and haven't figured out how, so it may be included. So as the schema.

I hope the next verson to be as good as the current, like Adonis itself.

**Comment by masan27** [comment]
Any news for this?
